readthedocs_theme/templates/includes/topnav.html

```diff
@@ -118,22 +147,34 @@
             {{ menu_main() }}
 
             <div class="right menu">
-              <div class="ui wide dropdown item" data-module="dropdown" data-action="select" data-display-type="block">
+
+              {# <div class="ui wide dropdown item" data-module="dropdown" data-action="select" data-display-type="block">
                 <span class="text">Help</span>
                 <i class="fad fa-caret-down icon"></i>
                 <div class="menu">
                   {{ menu_help() }}
                 </div>
-              </div>
+              </div> #}
+
               <div class="item">
 
-                <div class="ui fluid dropdown primary button" data-module="dropdown" data-action="select" data-display-type="block">
+                <div class="ui fluid dropdown button" data-module="dropdown" data-action="select" data-display-type="block">
                   <i class="fad fa-right-to-bracket icon"></i>
                   <span class="text">Log in</span>
                   <div class="menu">
                     {{ menu_log_in() }}
                   </div>
                 </div>
+              </div>
+
+              <div class="item">
+                <div class="ui fluid dropdown primary button" data-module="dropdown" data-action="select" data-display-type="block">
+                  <i class="fad fa-play icon"></i>
+                  <span class="text">Sign up</span>
+                  <div class="menu">
+                    {{ menu_sign_up() }}
```

Comment:
I wouldn't use a dropdown specifically for sign up links. All sign up links should trigger the sign up modal so that we have a chance to describe the two platforms to the user.